### ISSUE TITLE
Update the content of the tabs and how it's displayed

### ISSUE THREAD

**Issue by TTTaevas** [TTTaevas/taevas.xyz#8]
Some of the information in these tabs was written almost a year ago, around June 2023 if I'm not mistaken
That makes it outdated in placed, namely in the _About me_ tab, where I say I'm invested in cybersecurity and networking, when now those things are skillsets I essentially don't make use of anymore

The _My projects_ tab cannot really ever be outdated by design, but by coincidence, it also faces an issue I want to fix: It looks too different from other tabs
In other words, I have to check the looks of the content of all tabs, and update the content of some tabs

The four relevant tabs (so all the tabs currently available) are:

1. **About me** / À propos de moi
2. **My projects** / Mes projets
3. **Contact me** / Me contacter
4. **Support me** / Me soutenir

**Comment by TTTaevas** [comment]
Sounds reasonable to finally close this issue, totally not several months late btw
